src/pages/project/ProjectModal.tsx

```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from 'react-i18next';
+import { Modal } from '../../components/Modal';
+import { CristinProject, ResearchProject } from '../../types/project.types';
+import CreateProject from './project_wizard/CreateProject';
+
+interface ProjectModalProps {
+  isOpen: boolean;
+  toggleModal: () => void;
+  onProjectCreated: (value: CristinProject | ResearchProject) => void;
+}
+
+export const ProjectModal = ({ isOpen, toggleModal, onProjectCreated }: ProjectModalProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Modal
+      title={t('project.project_modal_title')}
```

Comment:
Får to a11y errors her pga manglende heading i dialogen. Ser ut til å la seg løse ganske enkelt ved å endre prop

```suggestion
      headingText={t('project.project_modal_title')}
```

![bilde](https://github.com/user-attachments/assets/e77963a3-7a83-4695-9952-20516089821c)

Reply:
Tar den heller bort

Reply:
Tror du må ha den? Jeg får fortsatt to feil for a11y etter testing med axe hvertfall (1 serious og 1 minor)